db/migrate/20241113182557_add_is_confidential_to_references.rb

```diff
@@ -0,0 +1,8 @@
+class AddIsConfidentialToReferences < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    add_column :references, :is_confidential, :boolean
+    add_index :references, :is_confidential, algorithm: :concurrently
```

Comment:
Do we need this to be indexed? if we aren't planning on doing a lot of searching based on this (eg. `Reference.where(is_confidential: true)`, I'm not sure it's necessary.

Reply:
It's unclear how the column will be used in the future. We have 546,975 entries on the references table which is only going to grow but I think you're right, this probably isn't needed at the moment and we can add it in the future if we find that we have a need for it.